debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlDatabaseSchema.java

```diff
@@ -269,7 +271,7 @@ else if (event instanceof SetVariableEvent) {
                     }
                 });
             }
-            else if (acceptableDatabase(databaseName)) {
+            else {
```

Comment:
What kind of event is this actually, with `ddlChanges` being empty?

Reply:
Well, that's good question but the code was present in the old connector too - I suppose it might be commands like `GRANT`